migrate/sqlMigrator/sqlMigrator_test.go

```diff
@@ -0,0 +1,183 @@
+		for _, exec := range *generateScriptExecutions(proj) {
+			err = dstore.InsertScriptExecution(&exec)
+			if err != nil {
+
+			}
+		}
+	}
+}
+
+func generateProject(projectGenCount int) (*model.InternalProject, *[]*model.TransactionTemplate, *[]*model.ScriptTemplate) {
+	proj := &model.InternalProject{
+		ID:                        uuid.New(),
+		UserID:                    uuid.New(),
+		Secret:                    uuid.New(),
+		PublicID:                  uuid.New(),
+		ParentID:                  nil,
+		Title:                     "Project number " + strconv.Itoa(projectGenCount),
+		Description:               "",
+		Readme:                    "",
+		Seed:                      0,
+		TransactionCount:          0,
+		TransactionExecutionCount: rand.Intn(5),
+		TransactionTemplateCount:  rand.Intn(5),
+		ScriptTemplateCount:       rand.Intn(5),
+		Persist:                   true,
+		CreatedAt:                 time.Time{},
+		UpdatedAt:                 time.Time{},
+		Version:                   build.Version(),
+	}
+
+	// Populate project with transaction templates
+	var ttpls []*model.TransactionTemplate
+	for i := 0; i < proj.TransactionTemplateCount; i++ {
+		ttpl := model.TransactionTemplate{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			Title:  "Transaction Template " + strconv.Itoa(i),
+			Index:  i,
+			Script: "Test script",
+		}
+
+		ttpls = append(ttpls, &ttpl)
+	}
+
+	// Populate project with script templates
+	var stpls []*model.ScriptTemplate
+	for i := 0; i < proj.ScriptTemplateCount; i++ {
+		stpl := model.ScriptTemplate{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			Title:  "Script Template " + strconv.Itoa(i),
+			Index:  i,
+			Script: "Test script",
+		}
+
+		stpls = append(stpls, &stpl)
+	}
+
+	return proj, &ttpls, &stpls
+}
+
+// generateAccounts generates a random number of accounts up to 10
+func generateAccounts(projID uuid.UUID) *[]*model.InternalAccount {
+	var accounts []*model.InternalAccount
+
+	for i := 0; i < rand.Intn(10); i++ {
+		// Generate address
+		bn := make([]byte, 8)
+		binary.BigEndian.PutUint32(bn, uint32(i))
+		addr := model.NewAddressFromBytes(bn)
+
+		account := model.InternalAccount{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: projID,
+			},
+			Address:   addr,
+			DraftCode: "test code " + strconv.Itoa(i),
+		}
+
+		accounts = append(accounts, &account)
+	}
+	return &accounts
+}
+
+func generateScriptExecutions(proj *model.InternalProject) *[]model.ScriptExecution {
+	var scriptExecs []model.ScriptExecution
+	for i := 0; i < proj.TransactionExecutionCount; i++ {
+		scriptExecs = append(scriptExecs, model.ScriptExecution{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			Index:     i,
+			Script:    "Test script execution",
+			Arguments: nil,
```

Comment:
lack of arguments, errors, and logs is problematic if you want just to use tests to prove functioning of this migration. Those values would in turn be migrated.